src/coreclr/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -6846,7 +6850,7 @@ HRESULT ProfToEEInterfaceImpl::SuspendRuntime()
     }
     CONTRACTL_END;
 
-    PROFILER_TO_CLR_ENTRYPOINT_SYNC_EX(
+    PROFILER_TO_CLR_ENTRYPOINT_ASYNC_EX(
```

Comment:
Since the ASYNC macro doesn't give us the same error checking on the k2PEETriggers flag it would be good to add the same check I recommended in EnumerateGCHeap objects here.

Reply:
Done